Dockerfile

```diff
@@ -3,11 +3,25 @@ FROM python:3.12
 RUN useradd -m -U app
 
 WORKDIR /home/app
```

Comment:
Consider using a different dir, such as `/opt` or just `/app`, to not interfere with bind mounts.